xgi/generators/random.py

```diff
@@ -20,15 +21,15 @@
 ]
 
 
-def random_hypergraph(N, ps, order=None, seed=None):
+def fast_random_hypergraph(n, ps, order=None, seed=None):
     """Generates a random hypergraph
 
     Generate N nodes, and connect any d+1 nodes
```

Comment:
add here or as a Note something about why this one is fast what the optimization is (or just a ref)

Reply:
Added a reference and a note to look at the refs.